<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PromptPlugin</class>
 <widget class="QWidget" name="PromptPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBox_UseFancyPrompts">
     <property name="text">
      <string>Use Fancy Prompts</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget_Prompt">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_FancyPrompts" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>387</width>
               <height>488</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QComboBox" name="comboBox_SelectFancyPrompt">
                <item>
                 <property name="text">
                  <string>Tiny</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Std</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Color</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Gentoo</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Dir</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Med</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Narrow</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Wide</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fancy</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Zee</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Date</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Curl</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_DisableUnicode">
                <property name="text">
                 <string>Disable Unicode</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_BoldLines">
                <property name="text">
                 <string>Use bold lines</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_LargerCompact">
                <property name="text">
                 <string>Make the larger prompts smaller</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_DoubledLines">
                <property name="text">
                 <string>Doubled Lines</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_MutedColors">
                <property name="text">
                 <string>Muted Colors</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_NoColors">
                <property name="text">
                 <string>No Colors</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_UseParens">
                <property name="text">
                 <string>Use Parentheses instead of Square Brackets</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Date Format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_DateFormat"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Time Format</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_TimeFormat"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Prompt Text</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_PromptText"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Extra lines before prompt</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_LinesBeforePrompt"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Leave so many right spaces</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_LeaveRightSpaces"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Centered Title</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_CenteredTitle"/>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QScrollArea" name="scrollArea_2">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>405</width>
            <height>506</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Themes</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_SelectTheme"/>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
